com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java

```diff
@@ -193,7 +193,7 @@ private void defineCopybook(String copybookName, Position position) {
       return;
     }
     if (checkThisCopybookNotPresentInHierarchy(copybookName)) {
-      semanticContext.getCopybooks().define(copybookName, position);
+      semanticContext.getCopybooks().addUsage(copybookName, position);
       semanticContext.getVariables().define(new Variable("-1", copybookName), position);
```

Comment:
It seems like the problem with the test caused by this line. I think `define()` here should be also replaced with `addUsage()`

Reply:
Also, try to replace definitions call in `CobolVariableContext#buildVariableStructure()` with one for usages

Reply:
It was not that, the problem was in `CobolSemanticParserImpl#processCopybooks`